src/models/users.js

```diff
@@ -0,0 +1,38 @@
+import { DataTypes } from "sequelize-cockroachdb";
+import sequelize from './index'
+
+const UserModel = sequelize.define(
+  "User",
+  {
+    email: {
```

Comment:
Email should be set as the primary key for users

Reply:
refactored the model to have email as primary key